packages/languages/bin/download.js

```diff
@@ -1,24 +1,21 @@
+/* eslint-disable no-console, no-process-exit, import/no-nodejs-modules */
+
 /**
  * External dependencies
  */
 const https = require( 'https' );
 const fs = require( 'fs' );
+const path = require( 'path' );
 
 const LANGUAGES_META_URL = 'https://widgets.wp.com/languages/calypso/languages-meta.json';
-const FILE_PATH = 'client/languages/languages-meta.json';
+const FILE_PATH = path.resolve( __dirname, '../src/languages-meta.json' );
```

Comment:
`path.join()` will give us the same result and is a bit more correct IMO

Reply:
[`56f0d23` (#46709)](https://github.com/Automattic/wp-calypso/pull/46709/commits/56f0d23af66b41528e943c53996fd010e6ec715f)